manager/assets/modext/widgets/core/modx.grid.js

```diff
@@ -715,6 +715,39 @@ Ext.extend(MODx.grid.Grid,Ext.grid.EditorGridPanel,{
     actionContextMenu: function(record, recordIndex, e) {
         this._showMenu(this, recordIndex, e);
     }
+
+    ,makeUrl: function () {
+        if (typeof this.config.urlFilters !== "undefined" && this.config.urlFilters !== null && this.config.urlFilters.length !== null && this.config.urlFilters.length > 0) {
```

Comment:
I believe this can be simplified to `if(Array.isArray(this.config.urlFilters) && this.config.urlFilters.length > 0)`

Reply:
You are right: https://caniuse.com/#search=isArray